src/main/java/org/jabref/gui/maintable/BibEntryTableViewModel.java

```diff
@@ -145,4 +145,8 @@ public ObservableValue<String> getFields(OrFields fields) {
     public StringProperty bibDatabaseContextProperty() {
```

Comment:
While you are on it, could you rename this to `bibDatabasePathProperty`? (Because it is not the full context, but "only" the path).